<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>841</width>
    <height>441</height>
   </rect>
  </property>
  <property name="acceptDrops">
   <bool>true</bool>
  </property>
  <property name="windowTitle">
   <string>Список</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout" columnstretch="1,10">
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <property name="spacing">
     <number>2</number>
    </property>
    <item row="0" column="0">
     <widget class="QTreeWidget" name="tree_widget">
      <property name="minimumSize">
       <size>
        <width>210</width>
        <height>0</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>Tahoma</family>
        <pointsize>8</pointsize>
       </font>
      </property>
      <property name="acceptDrops">
       <bool>false</bool>
      </property>
      <property name="editTriggers">
       <set>QAbstractItemView::NoEditTriggers</set>
      </property>
      <property name="dragEnabled">
       <bool>false</bool>
      </property>
      <property name="rootIsDecorated">
       <bool>true</bool>
      </property>
      <property name="uniformRowHeights">
       <bool>false</bool>
      </property>
      <property name="animated">
       <bool>true</bool>
      </property>
      <property name="allColumnsShowFocus">
       <bool>false</bool>
      </property>
      <property name="wordWrap">
       <bool>false</bool>
      </property>
      <property name="headerHidden">
       <bool>true</bool>
      </property>
      <column>
       <property name="text">
        <string>Категории</string>
       </property>
      </column>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QTableWidget" name="table_widget">
      <property name="font">
       <font>
        <family>Tahoma</family>
        <pointsize>8</pointsize>
       </font>
      </property>
      <property name="acceptDrops">
       <bool>false</bool>
      </property>
      <property name="editTriggers">
       <set>QAbstractItemView::NoEditTriggers</set>
      </property>
      <property name="dragEnabled">
       <bool>false</bool>
      </property>
      <property name="dragDropOverwriteMode">
       <bool>false</bool>
      </property>
      <property name="selectionBehavior">
       <enum>QAbstractItemView::SelectRows</enum>
      </property>
      <attribute name="verticalHeaderVisible">
       <bool>false</bool>
      </attribute>
      <attribute name="verticalHeaderDefaultSectionSize">
       <number>20</number>
      </attribute>
      <attribute name="verticalHeaderMinimumSectionSize">
       <number>18</number>
      </attribute>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="font">
    <font>
     <family>Tahoma</family>
     <pointsize>8</pointsize>
    </font>
   </property>
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);</string>
   </property>
   <property name="movable">
    <bool>false</bool>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="pb_tree_add"/>
   <addaction name="pb_tree_change"/>
   <addaction name="pb_tree_dell"/>
   <addaction name="action123"/>
   <addaction name="pb_table_add"/>
   <addaction name="pb_table_change"/>
   <addaction name="pb_table_dell"/>
   <addaction name="pb_table_filter"/>
   <addaction name="pb_table_transfer"/>
   <addaction name="pb_table_double"/>
   <addaction name="pb_table_update"/>
   <addaction name="pb_other"/>
   <addaction name="pb_print"/>
   <addaction name="pb_export"/>
  </widget>
  <action name="pb_tree_add">
   <property name="text">
    <string>Добавить</string>
   </property>
  </action>
  <action name="pb_tree_change">
   <property name="text">
    <string>Изменить</string>
   </property>
  </action>
  <action name="pb_tree_dell">
   <property name="text">
    <string>Удалить</string>
   </property>
  </action>
  <action name="action123">
   <property name="text">
    <string>'</string>
   </property>
  </action>
  <action name="pb_table_add">
   <property name="text">
    <string>Добавить</string>
   </property>
  </action>
  <action name="pb_table_change">
   <property name="text">
    <string>Изменить</string>
   </property>
  </action>
  <action name="pb_table_dell">
   <property name="text">
    <string>Удалить</string>
   </property>
  </action>
  <action name="pb_table_transfer">
   <property name="text">
    <string>Перенос</string>
   </property>
  </action>
  <action name="pb_table_double">
   <property name="text">
    <string>Дублирование</string>
   </property>
  </action>
  <action name="pb_table_filter">
   <property name="text">
    <string>Фильтр</string>
   </property>
  </action>
  <action name="pb_table_update">
   <property name="text">
    <string>Обновить</string>
   </property>
  </action>
  <action name="pb_other">
   <property name="text">
    <string>Разное</string>
   </property>
  </action>
  <action name="pb_export">
   <property name="text">
    <string>Экспорт</string>
   </property>
  </action>
  <action name="pb_print">
   <property name="text">
    <string>Печать</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>pb_tree_add</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>ui_add_tree_item()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>388</x>
     <y>220</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pb_tree_change</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>ui_change_tree_item()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>388</x>
     <y>220</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pb_tree_dell</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>ui_dell_tree_item()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>388</x>
     <y>220</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pb_table_add</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>ui_add_table_item()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>388</x>
     <y>220</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>tree_widget</sender>
   <signal>itemClicked(QTreeWidgetItem*,int)</signal>
   <receiver>MainWindow</receiver>
   <slot>ui_sorting()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>67</x>
     <y>128</y>
    </hint>
    <hint type="destinationlabel">
     <x>-45</x>
     <y>81</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pb_table_change</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>ui_change_table_item()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>388</x>
     <y>220</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pb_table_dell</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>ui_dell_table_item()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>388</x>
     <y>220</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>table_widget</sender>
   <signal>itemDoubleClicked(QTableWidgetItem*)</signal>
   <receiver>MainWindow</receiver>
   <slot>ui_double_click_table_item()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>538</x>
     <y>148</y>
    </hint>
    <hint type="destinationlabel">
     <x>952</x>
     <y>204</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pb_table_filter</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>ui_filter_table()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>388</x>
     <y>220</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pb_table_transfer</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>ui_transfer_table()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>388</x>
     <y>220</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pb_table_double</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>ui_double_item_table()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>388</x>
     <y>220</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pb_table_update</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>ui_update_table()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>388</x>
     <y>220</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pb_other</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>ui_other()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>388</x>
     <y>220</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pb_export</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>ui_export()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>401</x>
     <y>220</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pb_print</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>ui_print()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>420</x>
     <y>220</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>ui_add_tree_item()</slot>
  <slot>ui_change_tree_item()</slot>
  <slot>ui_dell_tree_item()</slot>
  <slot>ui_add_table_item()</slot>
  <slot>ui_sorting()</slot>
  <slot>ui_dell_table_item()</slot>
  <slot>ui_change_table_item()</slot>
  <slot>ui_double_click_table_item()</slot>
  <slot>ui_filter_table()</slot>
  <slot>ui_transfer_table()</slot>
  <slot>ui_double_item_table()</slot>
  <slot>ui_update_table()</slot>
  <slot>ui_other()</slot>
  <slot>ui_export()</slot>
  <slot>ui_print()</slot>
 </slots>
</ui>
